application/controllers/AuthenticationController.php

```diff
@@ -64,6 +102,14 @@ public function loginAction()
     public function logoutAction()
     {
         $auth = $this->Auth();
+        if (isset($_COOKIE['remember-me'])) {
```

Comment:
The "forget remember-me cookie code" must be moved down to the place where the code is executed if it's not an external user who is logging out.

Now you check the remember-me cookie even if the user is not authenticated or is an external user. Which is superfluous in both cases.